etc/cmake/options.cmake

```
@@ -161,6 +161,11 @@ if(OT_MAC_FILTER)
     list(APPEND OT_PRIVATE_DEFINES "OPENTHREAD_CONFIG_MAC_FILTER_ENABLE=1")
 endif()
 
+option(OT_MLE_LONG_ROUTES "enable MLE long routes extension")
```

Can we add a note here that this breaks Thread conformance, something like:
```suggestion
option(OT_MLE_LONG_ROUTES "enable MLE long routes extension (experimental, breaks Thread conformance)")
```

---

Good idea. Will do.

---

noted